dataTW.py

```diff
@@ -0,0 +1,67 @@
+import tweepy as tw
+from tweepy.auth import OAuthHandler
+import csv
+import urllib2
+import re
+import os
+from dotenv import load_dotenv, find_dotenv
+from config import tweetsId, tweets_user
+
+
+load_dotenv(find_dotenv())
+
+
+CONSUMER_KEY = os.getenv('CONSUMER_KEY')
+CONSUMER_SECRET = os.getenv('CONSUMER_SECRET')
+ACCESS_TOKEN = os.getenv('ACCESS_TOKEN')
+ACCESS_SECRET = os.getenv('ACCESS_SECRET')
+
+
+auth = tw.OAuthHandler(CONSUMER_KEY, CONSUMER_SECRET)
+auth.set_access_token(ACCESS_TOKEN, ACCESS_SECRET)
+
+api = tw.API(auth)
+
+
+def get_user_ids_of_post_likes(post_id):
+    try:
+        json_data = urllib2.urlopen(
```

Comment:
use requests instead of urllib2 (you already have it in the requirements). And use json/dicts instead of regular expressions